mitmproxy/utils/debug.py

```diff
@@ -5,18 +5,24 @@
 import signal
 import platform
 import traceback
+import subprocess
 
 from mitmproxy import version
 
 from OpenSSL import SSL
 
 
-def sysinfo():
+def dump_system_info():
+    old_dir = os.getcwd()
+    os.chdir(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
I think we can re-use this: https://github.com/mitmproxy/mitmproxy/blob/490872ebefd87e9a55f2db3dd18c0fbb58dbf206/mitmproxy/test/tutils.py#L22